api/tests/factory/build-skill-collection.js

```diff
@@ -0,0 +1,16 @@
+const faker = require('faker');
+const Skill = require('../../lib/domain/models/Skill');
+
+module.exports = function BuildSkillCollection({
+  name = faker.lorem.word(),
+  min = 3,
+  max = 5,
+} = {}) {
+  const collection = [];
+
+  for (let i = min; i <= max; i += 1) {
```

Comment:
Hello, je n'arrive pas à comprendre cette boucle for. 
Le seul objectif est de rajouter 3 skills à la collection ?
Je pense qu'on peut expliciter les deux variables min et max (min de quoi et max de quoi).

Reply:
okay, je l'ai compris en voyant son utilisation. Je pense qu'on peut quand même expliciter les variables min et max ou sortir les magic numbers dans des const.